tests/ui/specs/VirtualList/VirtualGridList/With5-way/VirtualGridList-GT-28584-specs.js

```diff
@@ -0,0 +1,53 @@
+const {expectFocusedItem} = require('../../VirtualList-utils');
+const Page = require('../VirtualGridListPage');
+
+describe('Navigate with 5-way', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should navigate from and to last item via 5-way [GT-28584]', function () {
+		// Step 3-1:  Knobs > VirtualGridList > dataSize > 17
+		Page.inputNumItems.moveTo();
+		Page.spotlightSelect();
+		Page.backSpace();
+		Page.backSpace();
+		Page.numPad(7);
+		Page.spotlightLeft();
+		// Step 4-1: 5-way Spot Image 16.
+		Page.item(Number(Page.bottomRightVisibleItemId().slice(4))).moveTo();
+		Page.spotlightDown();
+		// Step 4-2: Quickly 5-way Right.
+		Page.spotlightRight();
+		// Step 4-3: Quickly 5-way Down.
+		Page.spotlightDown();
+		// Step 4-3 Verify: Spotlight is on Image 16.
+		expectFocusedItem(16);
```

Comment:
It would be nice if there is a way to check if the previous item partially cut off.

Reply:
Suggested change.

Reply:
I checked you add for RTL only. Could you add it on LTR verification?

Reply:
I added it on LTR test.